.gitignore

```diff
@@ -1,2 +1,3 @@
 /.vscode
 /.lake
+.DS_Store
```

Comment:
Remove. Remove `/.vscode` too while you're at it. You can add those to `.git/info/exclude` if you want...